conda/environment-dev.yaml

```diff
@@ -0,0 +1,12 @@
+name: gc_derivatization-dev
+channels:
+  - conda-forge
+  - defaults
+dependencies:
+  - python >=3.8
+  - flake8
+  - mock
+  - pytest
+  - pytest-cov
+  - pytest-rerunfailures
+  - rdkit
```

Comment:
I think it would make sense to either have the dependencies in a `requirements.txt` or in the environment.yaml file, so similar as in MSME, we could actually use pip in the conda environment to install the dependencies? Especially in this package where we don't have that many dependencies.

Reply:
Done in 3269c7f and f8dd7ee.